WeightProducer/python/MCSampleValues.py

```diff
@@ -999,141 +817,164 @@ class MCSampleValuesHelper():
+        "ST_t-channel_antitop_5f_InclusiveDecays" : {
+            "CrossSection" : XSValues(
+                XS_13TeV=71.74, XSSource_13TeV="XSDB (NLO)",
+            ),
+        },
+        "ST_t-channel_top_5f_InclusiveDecays" : {
+            "CrossSection" : XSValues(
+                XS_13TeV=119.7, XSSource_13TeV="XSDB (NLO)",
+            ),
+        },
         "ST_tW_top_5f_NoFullyHadronicDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.09, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.91, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.91, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.45, XSSource_13TeV="XSDB (NLO)",
             ),
             "BranchingRatio" : BRValues(
                 BR_13TeV=(1-(1-3*0.105)**2), BRSource_13TeV="http://pdg.lbl.gov/2019/reviews/rpp2018-rev-top-quark.pdf",
             ),
         },
         "ST_tW_antitop_5f_NoFullyHadronicDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.06, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.97, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.97, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.51, XSSource_13TeV="XSDB (NLO)",
             ),
             "BranchingRatio" : BRValues(
                 BR_13TeV=(1-(1-3*0.105)**2), BRSource_13TeV="http://pdg.lbl.gov/2019/reviews/rpp2018-rev-top-quark.pdf",
             ),
         },
         "ST_tW_top_5f_inclusiveDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.09, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.91, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.91, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.45, XSSource_13TeV="GenXSecAnalyzer",
             ),
         },
         "ST_tW_antitop_5f_inclusiveDecays" : {
             "CrossSection" : XSValues(
-                XS_2016=38.06, XSSource_2016="XSDB (NLO)",
-                XS_2017=34.97, XSSource_2017="XSDB (NLO)",
-                XS_2018=34.97, XSSource_2018="GenXSecAnalyzer",
+                XS_13TeV=32.51, XSSource_13TeV="GenXSecAnalyzer",
+            ),
+        },
+        "tZq_ll_4f_ckm" : {
+            "CrossSection" : XSValues(
+                XS_13TeV=0.07561, XSSource_13TeV="XSDB (unknown)",
             ),
         },
+        #TODO
         "tZq_W_lept_Z_hadron_4f_ckm" : {
             "CrossSection" : XSValues(
                 XS_2016=0.1573, XSSource_13TeV="XSDB (unknown)",
             ),
         },
+        #TODO
         "tZq_Zhad_Wlept_4f_ckm" : {
             "CrossSection" : XSValues(
                 XS_2017=0.1518, XSSource_2017="XSDB (unknown)",
             ),
         },
         "WW" : {
             "CrossSection" : XSValues(
-                XS_13TeV=51.723, XSSource_13TeV="https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Diboson, WW>4q NNLO",
+                XS_13TeV=75.95, XSSource_13TeV="XSDB (unknown)",
```

Comment:
This doesn't seem right. WW is a well measured process. You should use the NNLO number unless you have some other compelling reason.

Reply:
Indeed, using the NNLO is better. As an aside, I see the number for WW -> 4Q had been used here, whereas I would think we want the inclusive number for "WW", or am I missing something ?

Reply:
The twiki doesn't have the inclusive cross section on it. The source they point to (https://arxiv.org/pdf/1408.5243.pdf) does, but I can't quite figure out how they got the numbers listed on the twiki. Maybe a more detailed reading would help.